### PR TITLE
[bn] Translate turnkey-solutions.md

### PR DESCRIPTION
**Pull request by nazma1234**
Translated ``turnkey-solutions.md`` for Bengali Documentation

Localizes ``content/bn/docs/setup/production-environment/turnkey-solutions.md``
issues: #31677

### CODE REVIEW

File: content/bn/docs/setup/production-environment/turnkey-solutions.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Turnkey ক্লাউড সমাধান
```
Comment (sftim) [inline content/bn/docs/setup/production-environment/turnkey-solutions.md:2]: If it's not obvious how to translate “Turnkey”, try starting from “Ready made” instead?

Comment (shannonxtreme) [reply content/bn/docs/setup/production-environment/turnkey-solutions.md:2]: +1 to Ready-made

Comment (mitul3737) [reply content/bn/docs/setup/production-environment/turnkey-solutions.md:2]: Okay! @sftim & @shannonxtreme

**Review by mitul3737 — APPROVED** [review]
/lgtm

**Pull request merged**